plutus-benchmark/default.tpl

```diff
@@ -0,0 +1,377 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
Is this supposed to be checked in? What is it? I guess a modification of some criterion template? Can we get some comments in the template saying which bits are changed?

Reply:
There is a README file in that directory...

I'll add a comment in the file itself though.

Reply:
I think you still need the other ones to be checked in to support some of Criterion's other command-line options.